src/vaadin-grid-event-context-mixin.html

```diff
@@ -0,0 +1,60 @@
+<!--
+@license
+Copyright (c) 2018 Vaadin Ltd.
+This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+-->
+
+<script>
+  window.Vaadin = window.Vaadin || {};
+  window.Vaadin.Grid = window.Vaadin.Grid || {};
+
+  /**
+   * @polymerMixin
+   */
+  Vaadin.Grid.EventContextMixin = superClass => class EventContextMixin extends superClass {
+
+    /**
+     * Returns an object with context information about the event target:
+     * - `item`: the data object corresponding to the targeted row (not specified when targeting header or footer)
+     * - `column`: the column element corresponding to the targeted cell (not specified when targeting row details)
+     * - `section`: whether the event targeted the body, header, footer or details of the grid
+     *
+     * These additional properties are included when `item` is specified:
+     * - `index`: the index of the item
+     * - `selected`: the selected state of the item
+     * - `detailsOpened`: whether the row details are open for the item
+     * - `expanded`: the expanded state of the tree toggle
+     * - `level`: the tree hierarchy level represented with a horizontal offset of the toggle button
```

Comment:
Should not mention "toggle button". This only applies in the `vaadin-tree-toggle`

Reply:
I copied that from the `renderer()` docs. Done.

Reply:
Ouch, shouldn't be there either